### PR TITLE
NotFoundError added

### LINKED ISSUE #132: Checking `foreignKey` value if exists on the table in related queries
Let's assume that we have a pagination URL like this;

```bash
http://localhost:3000/api/users/66/posts
```

Currently, Axe API doesn't check whether the user id (`66`) exists on the table. This query will be returned empty anyway. But, it should throw 404.

~> Devs should be able send `ignoreParentId`~

> Soft-delete should be supported by the model's soft-delete field

### PR DESCRIPTION
**Pull request by Arifkarakilic**
ShowHandler item control status code changed and error class default value bad request made 
Close #132

### CODE REVIEW

File: src/Handlers/ShowHandler.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { HandlerTypes, HookFunctionTypes } from "../Enums";
 import ApiError from "../Exceptions/ApiError";
 import { QueryService } from "../Services";
 import { Knex } from "knex";
+import { StatusCodes } from "http-status-codes";
```
Comment (ozziest) [inline src/Handlers/ShowHandler.ts:14]: I remember that I already removed the `http-status-codes` package from dependencies. You can find the `StatusCodes` enum in the `Enum.ts` file. Can you use it?

**Review by ozziest — CHANGES_REQUESTED** [review]
This is a great start, @Arifkarakilic! 👏 🥳 

I requested some changes. Let's keep coding little bit more. 💪

**Comment by Arifkarakilic** [comment]
done

File: src/Handlers/ShowHandler.ts [hunk 2]
```diff
@@ -50,9 +51,10 @@ export default async (pack: IRequestPack) => {
 
   let item = await query.first();
   if (!item) {
-    throw new ApiError(`The item is not found on ${model.name}.`);
-  }
-
+    throw new ApiError(`The item is not found on ${model.name}.`,StatusCodes.NOT_FOUND);
+  }//parentı kontrol edip gerçekten varsa 404le item not found döndermemiz gerekiyor
```
Comment (ozziest) [inline src/Handlers/ShowHandler.ts:55]:
```suggestion
  }
```

**Review by ozziest — CHANGES_REQUESTED** [review]
Congrats! 👏 👏 It looks great. 
But we need two more things;
- Integration tests: we need to check if really got 404
- Changelog change: we need to increase the version and update changelog file

**Review by ozziest — CHANGES_REQUESTED** [review]
We don't need to create a new integration test file. If you execute the following command;

`npm run test:mysql8`

then you can see the following error;

image

So we should update that test file with the correct HTTP response code.

**Review by ozziest — CHANGES_REQUESTED** [review]
We need to fix CHANGELOG. ☺️

**Review by ozziest — APPROVED** [review]

**Pull request merged**